concordia/views.py

```diff
@@ -1051,9 +1051,16 @@ def submit_transcription(request, *, pk):
     transcription = get_object_or_404(Transcription, pk=pk)
 
     if (
-        transcription.submitted
-        or transcription.asset.transcription_set.filter(supersedes=pk).exists()
-    ):
+        transcription.submitted and not transcription.rejected
+    ) or transcription.asset.transcription_set.filter(supersedes=pk).exists():
```

Comment:
I'm wondering whether it's worth splitting these tests out more since it's a bunch of conditions to parse reading this block.

Reply:
Not sure what you mean by that. Can you elaborate?

Reply:
I was just thinking that the layout was a little ungainly with the big parenthesized expression followed by an `or` and was wondering whether that would be worth assigning a temporary variable for clarity (e.g. `if (transcription.submitted and not transcription.rejected) or is_superseded:`

Reply:
Oh right on. Yeah that makes sense.